Scripts/import-logs.ps1

```diff
@@ -17,107 +17,61 @@ $region = 'us-east-1'
 $s3BaseKey =  $Lansaversion +'/'+ $PipelineReleaseId
 
 
+# Mapping the database to root folder
+# Here DB2ISERIES and MSSQLS are not included as DB2ISERIES is not mapped to any root path and MSSQLS is mapped to LANSA which is included by default
+$DB_ROOT_Map = @{
 
-
-Write-Host($db)
-Write-Host('')
+    SQLAZURE = "AZURESQL"
+    SQLANYWHERE = "SQLANYWHERE"
+    MYSQL = "MYSQL"
+    ODBCORACLE = "ORACLE"
+}
 $databases = New-Object -TypeName 'System.Collections.ArrayList';
 Write-Host('list of databases:') 
 Write-Host('--------------------------------------------------------------')
 foreach ($db in $databaseTypes)
 {
-    if($db -ne 'SQL Server')
-    {
-        $databases.Add($db)
-    }  
+    
+    if($db -ne 'DB2ISERIES'){
+        $databases.Add($DB_ROOT_Map[$db])
+    }     
     Write-Host($db) 
```

Comment:
Log both $db and $DB_ROOT_Map[$db]. The latter is far more important. Its what the code is deriving.